docs/projects/variables/worker-pool-variables.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Worker Pool Variables
+description: Create a worker pool variable and use it in deployment steps
+position: 60
+---
+
+Steps that use workers can specify a worker pool directly on the step or have the step depend on a worker pool variable.  First you should set up your [worker](/docs/infrastructure/workers/index.md) and [worker pool](/docs/infrastructure/workers/worker-pools.md) infrastructure.
+
+In the variable editor select **Change Type** and **Worker Pool** as the variable type.  The **Add Variable** window is then displayed and lists all the available worker pools.
```

Comment:
Wondering if it's clear enough that you need "open editor" to change the type and assign the pool?

Reply:
The option is available directly on the grid
![image](https://user-images.githubusercontent.com/1687639/75129384-74471780-5714-11ea-97b7-cdb763e6453c.png)